opening_hours/utils/hauki_api_client.py

```diff
@@ -14,18 +21,84 @@
 
 
 class HaukiAPIClient:
-    """Hauki API client, minimal functionality implemented."""
-
-    @staticmethod
-    def build_url(endpoint: Literal["resource", "opening_hours", "date_period"], resource_id: str | None = None) -> str:
+    def __init__(self):
         if not settings.HAUKI_API_URL:
             raise HaukiConfigurationError("HAUKI_API_URL environment variable must to be configured.")
 
-        url = urljoin(settings.HAUKI_API_URL, f"/v1/{endpoint}/")
-        if resource_id:
-            url += f"{resource_id}/"
+    ############
+    # resource #
+    ############
+
+    def get_resources(self, *, hauki_resource_ids: list[int], **kwargs: Any) -> HaukiAPIResourceListResponse:
```

Comment:
Could you document that `**kwargs` can be? Can be either with a TypedDict:

```python
_(..., **kwargs):
    kwargs: ResourceExtraParams
```

... or a link to the appropriate documentation on the Hauki side.